src/main/main.js

```diff
@@ -343,3 +348,68 @@ function handleSquirrelEvent() {
 	}
 	log.info('end handleSquirrelEvent');
 }
+
+function startStakingTest(ctx) {
```

Comment:
@sk91 curious, why not run this test with Jest?

Reply:
@rodrigopavezi it is an integration test, not unit test. Only for dev purposes right now.
Test like this will be run in the new blockchain - integration test suite that is currently WIP.